variables.tf

```diff
@@ -132,3 +132,8 @@ variable "service_label" {
   type        = string
 }
 
+variable "connection_draining_timeout_sec" {
+  description = "Time for which instance will be drained"
+  default     = "0"
```

Comment:
We should default to `null` to inherit the provider default.

Reply:
I will fix the variable. As far as the make generate_docs this is what I see as I do that...what am I missing?

` make generate_docs
docker run --rm -it \
		-v /Users/andrew.strozyk/github/terraform-google-lb-internal:/workspace \
		gcr.io/cloud-foundation-cicd/cft/developer-tools:0.6.0 \
		/bin/bash -c 'source /usr/local/bin/task_helper_functions.sh && generate_docs'
Generating markdown docs with terraform-docs
Working in . ...
Success!
Skipping ./examples/minimal because README.md does not exist.
Working in ./examples/simple ...
Success!
Skipping ./test/fixtures/minimal because README.md does not exist.
Skipping ./test/setup because README.md does not exist.`

Reply:
That looks like what it's supposed to do.

Reply:
Okay cool. So is this good then? Thanks